pkg/apis/componentconfig/types_pluginargs.go

```diff
@@ -131,3 +133,19 @@ type HighNodeUtilizationArgs struct {
 	Thresholds    api.ResourceThresholds
 	NumberOfNodes int
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// DefaultEvictorArgs holds arguments used to configure DefaultEvictorArgs plugin.
+type DefaultEvictorArgs struct {
+	metav1.TypeMeta
+
+	Nodes                   []*v1.Node
+	EvictLocalStoragePods   bool
+	EvictSystemCriticalPods bool
+	IgnorePvcPods           bool
+	EvictFailedBarePods     bool
+	PriorityThresholds      *int32
+	NodeFit                 bool
+	LabelSelector           labels.Selector
```

Comment:
Worth noting `LabelSelector` is currently not wired and have not been for some time.  Nevertheless, it will be used by yet-to-be-implemented pre-eviction-filter to exclude pods from eviction.

Reply:
Taking note :spiral_notepad: ;)